.github/workflows/appengine_deploy.yml

```diff
@@ -0,0 +1,54 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Deploy to App Engine
+
+# Controls when the workflow will run
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  prepare:
+    name: Prepare
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Prepare demo files
+        run: |
+          npm install
```

Comment:
I believe the reason we need to `npm install` is because we need the dependencies for the advanced playground to be installed so we can copy them over.

We could optionally add a comment here about that because it's something we've forgotten to do in the past.

Reply:
Updated.